libs/jwt-cpp/Makefile

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright © 2025 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=jwt-cpp
+PKG_VERSION:=0.7.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/Thalhammer/jwt-cpp/releases/download/v$(PKG_VERSION)/
+PKG_HASH:=d45894f57437ce45233cfe0e07383c4e1f32f969edfd8df8347e177b13bf74e5
+
+PKG_MAINTAINER:=David Woodhouse <dwmw2@infradead.org>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+CMAKE_INSTALL:=1
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+TAR_CMD=$(HOST_TAR) -C $(1) $(TAR_OPTIONS)
+
+
+#CMAKE_OPTIONS += \
+
+define Package/jwt-cpp
+  BUILDONLY:=1
+  SECTION:=devel
+  CATEGORY:=Development
+  SUBMENU:=Libraries
+  TITLE:=jwt-cpp is a header only library for creating and validating JSON Web Tokens in C++11
+  URL:=https://github.com/Thalhammer/jwt-cpp
+endef
+
```

Comment:
This is missing install section or rather InstallDev.

Reply:
Doesn't CMAKE_INSTALL do that? This is mostly just copied from libs/cereal/Makefile which is a similar build-only library.